src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs

```diff
@@ -196,6 +196,12 @@ private void ProcessRequests(ITestRequestManager testRequestManager)
                                 break;
                             }
 
+                        case MessageType.CancelDiscovery:
```

Comment:
We'll need to add acceptance tests for this.

Reply:
You mean targeting at the vstestconsole wrapper layer ?

Reply:
Yes.

Reply:
Added the acceptance test